### ISSUE TITLE
Random test failure (can_solve_with_solution_interpolation)

### ISSUE THREAD

**Issue by thofma** [Nemocas/AbstractAlgebra.jl#766]
Seen on CI. Here is the stacktrace:
```julia
Generic.Mat.can_solve_with_solution_interpolation: Error During Test at D:\a\AbstractAlgebra.jl\AbstractAlgebra.jl\test\generic\Matrix-test.jl:2104
  Got exception outside of a @test
  Not enough interpolation points in ring
  Stacktrace:
   [1] error(::String) at .\error.jl:33
   [2] can_solve_with_solution_interpolation_inner(::AbstractAlgebra.Generic.MatSpaceElem{AbstractAlgebra.Generic.Poly{AbstractAlgebra.Generic.Res{BigInt}}}, ::AbstractAlgebra.Generic.MatSpaceElem{AbstractAlgebra.Generic.Poly{AbstractAlgebra.Generic.Res{BigInt}}}) at D:\a\AbstractAlgebra.jl\AbstractAlgebra.jl\src\generic\Matrix.jl:2474
   [3] can_solve_with_solution_interpolation(::AbstractAlgebra.Generic.MatSpaceElem{AbstractAlgebra.Generic.Poly{AbstractAlgebra.Generic.Res{BigInt}}}, ::AbstractAlgebra.Generic.MatSpaceElem{AbstractAlgebra.Generic.Poly{AbstractAlgebra.Generic.Res{BigInt}}}) at D:\a\AbstractAlgebra.jl\AbstractAlgebra.jl\src\generic\Matrix.jl:2580
   [4] top-level scope at D:\a\AbstractAlgebra.jl\AbstractAlgebra.jl\test\generic\Matrix-test.jl:2132
   [5] top-level scope at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\Test\src\Test.jl:1115
   [6] top-level scope at D:\a\AbstractAlgebra.jl\AbstractAlgebra.jl\test\generic\Matrix-test.jl:2105
   [7] include(::String) at .\client.jl:457
   [8] top-level scope at D:\a\AbstractAlgebra.jl\AbstractAlgebra.jl\test\Rings-test.jl:12
   [9] include(::String) at .\client.jl:457
   [10] top-level scope at D:\a\AbstractAlgebra.jl\AbstractAlgebra.jl\test\AbstractAlgebra-test.jl:3
   [11] include(::String) at .\client.jl:457
   [12] top-level scope at D:\a\AbstractAlgebra.jl\AbstractAlgebra.jl\test\runtests.jl:113
   [13] include(::String) at .\client.jl:457
   [14] top-level scope at none:6
   [15] eval(::Module, ::Any) at .\boot.jl:331
   [16] exec_options(::Base.JLOptions) at .\client.jl:272
   [17] _start() at .\client.jl:506
  
Test Summary:                                     | Pass  Error  Total
Generic.Mat.can_solve_with_solution_interpolation |   43      1     44
ERROR: LoadError: LoadError: LoadError: LoadError: Some tests did not pass: 43 passed, 0 failed, 1 errored, 0 broken.
in expression starting at D:\a\AbstractAlgebra.jl\AbstractAlgebra.jl\test\generic\Matrix-test.jl:2104
in expression starting at D:\a\AbstractAlgebra.jl\AbstractAlgebra.jl\test\Rings-test.jl:12
in expression starting at D:\a\AbstractAlgebra.jl\AbstractAlgebra.jl\test\AbstractAlgebra-test.jl:3
in expression starting at D:\a\AbstractAlgebra.jl\AbstractAlgebra.jl\test\runtests.jl:113
ERROR: Package AbstractAlgebra errored during testing
Stacktrace:
 [1] pkgerror(::String) at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\Pkg\src\Types.jl:52
 [2] test(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}; coverage::Bool, julia_args::Cmd, test_args::Cmd, test_fn::Nothing) at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\Pkg\src\Operations.jl:1578
 [3] test(::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}; coverage::Bool, test_fn::Nothing, julia_args::Cmd, test_args::Cmd, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\Pkg\src\API.jl:327
 [4] #test#61 at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\Pkg\src\API.jl:67 [inlined]
 [5] test(; name::Nothing, uuid::Nothing, version::Nothing, url::Nothing, rev::Nothing, path::Nothing, mode::Pkg.Types.PackageMode, subdir::Nothing, kwargs::Base.Iterators.Pairs{Symbol,Bool,Tuple{Symbol},NamedTuple{(:coverage,),Tuple{Bool}}}) at C:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.5\Pkg\src\API.jl:80
 [6] top-level scope at none:1
```

**Comment by wbhart** [comment]
Yeah I guess that can fail. We may need to up the characteristic to prevent it being an irritation.

**Comment by wbhart** [comment]
Duplicate of #788 which is fixed by #825
